app/views/layouts/application.html.erb

```diff
@@ -39,6 +39,10 @@
       } %>
     <% end %>
 
+    <% if flash["confirmation"] %>
```

Comment:
Perhaps we can give this a generic name so we can use it here too:

https://github.com/alphagov/content-publisher/blob/37a87352be980f27e56aeb5a17b837bd18bbc80e/app/views/documents/show/_summary_tab.html.erb#L3-L5

Reply:
Can do, although that would make it appear at the top instead of below the title. That OK?

Reply:
Discussed offline - we'll leave it where it is for now.